PORTAL/jobs/_workers.py

```diff
@@ -122,19 +127,21 @@ class Workers:
 
     @classmethod
     def from_config(cls, cfg, JobsFS=WorkerJobsFS):
-        worker = Worker.from_config(cfg.worker, JobsFS)
-        return cls(worker)
+        workers = {}
+        for worker_name, worker in cfg.items():
```

Comment:
If we're passing in the set of worker configs then it might make more sense to call this "from_configs".  That said, my inclination would be to pass the whole config in:

```python
    def from_config(cls, cfg, JobsFS=WorkerJobsFS):
        workers = {}
        for worker_name, worker in cfg.workers.items():
```